src/10_functions.py

```diff
@@ -2,11 +2,24 @@
 
 # YOUR CODE HERE
 
+
+def check_if_even(num):
+    if num % 2 == 0:
+        return True
```

Comment:
Just `return n % 2 == 0` would be enough. It will return True or False based on this line implicitly.